### PR TITLE
Feature/octane

### PR DESCRIPTION
**Pull request by Matthijsy**
### Summary
Time for even more octange changes! This time it is time for the models. 

I also tried to cleanup the models a bit. The only models that could use some more cleanup is the form models, but those are quite complicated and probably need some refactor all together, so will do that later on.

### CODE REVIEW

File: app/models/activity.js [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
+  @attr coverPhoto;
+  @attr coverPhotoUrl;
+
+  // Relations
+  @belongsTo('user') author;
+  @belongsTo('group') group;
+  @belongsTo('form/form') form;
 
-  // Computed properties
-  formattedPrice: computed('price', function() {
```
Comment (wilco375) [inline app/models/activity.js:27]: Why is the price not formatted to two decimals anymore?

File: app/models/forum/post.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+import Model, { belongsTo, attr } from '@ember-data/model';
 
-export default Model.extend({
+export default class Post extends Model {
   // Properties
-  message: attr('string'),
-  messageCamofied: attr('string'),
-  createdAt: attr('date'),
-  updatedAt: attr('date'),
+  // @attr message;
```
Comment (wilco375) [inline app/models/forum/post.js:6]: Why is this attribute commented out?

File: app/models/group.js [hunk 3]
```diff
@@ -33,16 +31,21 @@ export default Model.extend(AvatarModelMixin, {
...
 
     return null;
-  }),
-  alternativeGroupLogoText: computed('name', function() {
-    return `Het logo van ${this.name}`;
-  }),
+  }
+
+  get avatarUrlOrDefault() {
+    return this.avatarUrl || '/images/fallback/group_avatar_default.png';
```
Comment (wilco375) [inline app/models/group.js:37]: Why did you not make this a constant like the other default images?

File: app/models/forum/post.js [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
   // Relations
-  author: belongsTo('user'),
-  thread: belongsTo('forum/thread'),
+  @belongsTo('user') author;
+
+  @belongsTo('forum/thread') thread;
 
   // Computed properties
-  isUpdated: computed('createdAt', 'updatedAt', function() {
+  @computed('createdAt', 'updatedAt')
```
Comment (wilco375) [inline app/models/forum/post.js:17]: (Why) do you need to explicitly give these variables here? Isn't it the same as, for example, `userIsCurrentlyMember` in membership.js?

File: app/models/group.js (hunk 3 above)
Comment (Matthijsy) [reply app/models/group.js:37]: because this image is only used for groups, the other are (as far as I remember) used at multiple locations

**Review by wilco375 — APPROVED** [review]

**Pull request merged**